flake.nix

```diff
@@ -0,0 +1,36 @@
+{
+  description = "Ephemeral IAM";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-22.05";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+    let
+      pkgs = nixpkgs.legacyPackages.${system};
+     in {
+        packages.default = pkgs.buildGoModule rec {
+          pname = "ephemeral-iam";
+          version = "0.0.22";
```

Comment:
what's the process for releasing a new version? do we need to bump this number too in addition of creating a new git tag?

can that be written in a `RELEASING.md` file?

Reply:
:/ yeah this will need to be manually bumped. We could pull a pid1 and do `0.0.0-${git-sha-here}`

Reply:
that could work!

Reply:
Sticking with version number for now and added a `RELEASING.md` file